src/api/queries/arbeidsgiver/oppfolgingsplanerQueriesAG.ts

```diff
@@ -35,6 +35,17 @@ export const useOppfolgingsplanerAG = () => {
   );
 };
 
+export const useAktivPlanAG = (): Oppfolgingsplan | undefined => {
```

Comment:
Jeg tror @Vitenok har en liknende hook i sin pr, så vi får passe på å ikke få to nesten helt like.